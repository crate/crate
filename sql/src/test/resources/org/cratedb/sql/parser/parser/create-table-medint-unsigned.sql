CREATE TABLE t (c mediumint unsigned)