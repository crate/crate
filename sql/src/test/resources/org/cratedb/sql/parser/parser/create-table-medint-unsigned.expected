org.cratedb.sql.parser.parser.CreateTableNode@354742ee
name: t
statementType: CREATE TABLE
lockGranularity: R
existenceCheck: NO_CONDITION
tableElementList: 	
	org.cratedb.sql.parser.parser.TableElementList@597b3295

	[0]:	
	org.cratedb.sql.parser.parser.ColumnDefinitionNode@27360218
	type: MEDIUMINT UNSIGNED
	name: c
	elementType: null